Gopkg.toml

```diff
@@ -0,0 +1,34 @@
+# Gopkg.toml example
+#
+# Refer to https://golang.github.io/dep/docs/Gopkg.toml.html
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
```

Comment:
Is this commented out? Should we keep it?

Reply:
@hundredblocks That's automatically generated by the go dependency manager... I think to use as an example? Should be fine to leave